pyproject.toml

```diff
@@ -0,0 +1,57 @@
+[build-system]
+requires = ["setuptools>=68.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools]
+platforms = ["Linux x86, x86-64"]
+
+[tool.setuptools.packages.find]
+include = ["sopel_slap", "sopel_slap.*"]
+namespaces = false
+
+[project]
+name = "sopel-slap"
+version = "0.2.0.dev0"
+description = "Plugin for Sopel that lets users slap each other in fun ways"
+keywords = [
+  "sopel",
+  "plugin",
+  "slap",
+  "pranks",
+  "bot",
+  "irc",
+]
+maintainers = [
+  { name="dgw", email="dgw@technobabbl.es" },
+]
+authors = [
+  { name="dgw", email="dgw@technobabbl.es" },
+  { name="Florian Strzelecki", email="florian.strzelecki@gmail.com" },
+]
+
+readme = "README.md"
+license = { text="EFL-2.0" }
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "Intended Audience :: System Administrators",
+    "License :: Eiffel Forum License (EFL)",
+    "License :: OSI Approved :: Eiffel Forum License",
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
```

Comment:
Where's 3.11-3.12? It should work on those, assuming it's installed under a compatible Sopel version.

Reply:
Untested so far. I've the same issue with other plugins, technically they should work fine with those Python versions, but since Sopel doesn't yet...

Reply:
I've added them since Sopel 8 is out!